src/common/values/constant-value.ts

```diff
@@ -4,3 +4,13 @@ export const RPC_URI = process.env.NEXT_PUBLIC_RPC_URI ?? '';
 export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_TRACKING_ID ?? '';
 
 export const DAY_TIME = 86_400_000 as const; // Day time: 24 * 60 * 60 * 1000
+
+export enum ChainType {
+  TESTNET5 = 'test5',
+  TESTNET4 = 'test4',
```

Comment:
We can depreciate all test4 info

Reply:
I removed it this commit (https://github.com/onbloc/gnoscan/pull/121/commits/a4a2dbdefaeaf1992946512a603abb75d374ad08)